microcosm_eventsource/stores/rollup.py

```
@@ -88,6 +88,13 @@ def search(self, **kwargs):
             for row in self._search_query(aggregate, **kwargs).all()
         ]
 
+    def search_first(self, **kwargs):
```

maybe we can use the existing `search` function with `limit=1`?

---

that would return an array

---

and add `res[0]`

---

which will throw an exception instead of returning None